docs/sources/tasks/debug.md

```diff
@@ -70,12 +73,33 @@ The clustering page shows the following information for each cluster node:
 * The node's current state (Viewer/Participant/Terminating).
 * The local node that serves the UI.
 
+### Live Debugging page
+
+<!-- TODO: add screenshot once the branch is ready to go to main -->
+
+Live debugging provides a real-time stream of debugging data from a component. You can access this page from the corresponding [Component detail page](#component-detail-page).
+
+Live debugging allows you to do the following:
+
+* Pause and clear the data stream.
+* Sample data and disable auto-scrolling to handle heavy loads.
+* Search through the data using keywords.
+* Copy the entire data stream to the clipboard.
+
+The format and content of the debugging data vary depending on the component type.
+
+{{< admonition type="note" >}}
+Live debugging is not yet available in all components.
```

Comment:
That's a good point. I did it like this because I thought that the list could grow quickly and be annoying to keep up to date. But it might still be better than letting users try and be disappointed

Reply:
I think a good future improvement would be for the "Live debugging" button to only show when a component supports it. Can we track that in a ticket?

Reply:
Is this something that will be added one component at a time... or batches of components?  Is there a pattern to what is available now, and what will be added later?  eg all prometheus.* first, then all loki.* next and so on.

We could start with a list now if it's short and adapt to something else later?

Reply:
There will be batches of components because for most it will be added on common codes. For example, adding all otel.processor.* will be done in one go. But the problem is that there can be exceptions and there is a risk that the list is unsync.

Reply:
But that could be good enough for a start and Robert's proposal is reasonable and should not be too hard to implement

Reply:
We can open an issue to automate updating the docs? We already have similar processes in place to update docs in an automated way.

Reply:
It's going to be a pretty long list at some point if we name all components that have the feature enabled... automated or not. :thinking:

Reply:
Not a super big fan of going the extra step to automate the doc on that. Can we start with a manual list, group the components as @clayton-cornell suggested and once we have the support for quite a few components we get rid of the list? In the meantime, we implement @rfratto's suggestion to show only the button on components that support it?

Reply:
https://github.com/grafana/alloy/issues/1008